diff --git a/src/lib/File.luau b/src/lib/File.luau
index 5966d6f..007165e 100644
--- a/src/lib/File.luau
+++ b/src/lib/File.luau
@@ -64,7 +64,7 @@ function File.create(filePath, initialContents, overwrite)
 
 	if fs.isFile(filePath) then
 		if overwrite then
-			fs.removeDir(filePath)
+			fs.removeFile(filePath)
 		else
 			error(`Cannot create a new file. A file already exists at that path({filePath})`)
 		end
